sym-lib-table

```diff
@@ -5,4 +5,5 @@
   (lib (name vnq7140aj)(type Legacy)(uri ${KIPRJMOD}/vnq7140aj.lib)(options "")(descr ""))
   (lib (name tle9471)(type Legacy)(uri ${KIPRJMOD}/tle9471.lib)(options "")(descr ""))
   (lib (name max9924)(type Legacy)(uri ${KIPRJMOD}/max9924.lib)(options "")(descr ""))
+  (lib (name proteus-rescue)(type Legacy)(uri ${KIPRJMOD}/proteus-rescue.lib)(options "")(descr ""))
```

Comment:
this shouldn't be here - you may need to manually remove it after running `git submodule update --init`